data/functions/itemized.sql

```diff
@@ -16,3 +124,22 @@ begin
     perform retry_processing_schedule_b_records(:START_YEAR_AGGREGATE);
 end
 $$ language plpgsql;
+
+CREATE OR REPLACE FUNCTION rename_table_cascade(table_name TEXT) RETURNS VOID AS $$
+DECLARE
+    child_tmp_table_name TEXT;
+    child_tables_cursor CURSOR (parent TEXT) FOR
+        SELECT c.oid::pg_catalog.regclass AS name
+        FROM pg_catalog.pg_class c, pg_catalog.pg_inherits i
+        WHERE c.oid=i.inhrelid
+            AND i.inhparent = (SELECT oid from pg_catalog.pg_class rc where relname = parent);
+BEGIN
+    EXECUTE format('DROP TABLE IF EXISTS %I CASCADE', table_name);
+    EXECUTE format('ALTER TABLE %1$I_tmp RENAME TO %1$I', table_name);
```

Comment:
`%1$I` allows you to use the same positional parameter (in this case, the first) for multiple substitutions. Otherwise, I would have had to write:
```
EXECUTE format('ALTER TABLE %I_tmp RENAME TO %I', table_name, table_name);
```